deli_counter.rb

```diff
@@ -0,0 +1,46 @@
+# Anshul Arora
+katz_deli = []
+$current_line = []
+
+def line(arr)
+  if arr.empty?
+    puts "The line is currently empty." 
+  else
+    $current_line.clear
```

Comment:
dont make use of `clear` here. simply reassign `current_line` to a new array as such `$current_line = []`

Reply:
the reason you needed to use `clear` is a side effect of  `current_line` being global. you are clearing it because there is a chance it has a value from a previous run (since it is global). This is just extra overhead for no benefit. `current_line` should be passed into these methods so these issues do not happen. This is a perfect example of what i mentioned earlier where globals can cause unexpected behaviors and are almost always a bad idea. There's usually always a way to safely scope variable where they are used only